swift-ios/Task Tracker/TasksViewController.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  ProjectViewController.swift
+//  Task Tracker
+//
+//  Created by MongoDB on 2020-05-07.
+//  Copyright © 2020 MongoDB, Inc. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class TasksViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    let project: Project?
+    let partitionValue: String
+    let realm: Realm
+    let tasks: Results<Task>
+    let tableView = UITableView()
+    var notificationToken: NotificationToken?
+
+    required init(project: Project?) {
+        guard let user = app.currentUser() else {
+            fatalError("Must be logged in to access this view")
+        }
+
+        self.project = project
+
+        // For tutorial purposes, we can pass nil as the project.
+        // This allows us to work with the Tasks of a default project, which
+        // will use the hardcoded ID "My Project".
+        self.partitionValue = project != nil ? "\(project!._id)" : "My Project"
```

Comment:
constant?

Reply:
You mean for "My Project"? I don't know... for use in the one place and as a tutorial purposes only thing, I feel that would be more confusing than helpful...